profile/README.md

```diff
@@ -0,0 +1,41 @@
+![Foreman Header](https://raw.githubusercontent.com/theforeman/.github/main/profile/foreman-header.png)
+
+# The Foreman Project
+
+Foreman is a complete lifecycle management tool for physical and virtual servers. We give system administrators the power to easily automate repetitive tasks, quickly deploy applications, and proactively manage servers, on-premise or in the cloud.
+
+## Features
+
+### Provisioning
+
+Provision on bare-metal (MaaS) & public or private clouds all from one place with one simple process.
+
+### Configuration
+
+A complete configuration management solution including an ENC for Puppet and Salt, built-in support for parameterized classes and hierarchical parameter storage. Or integrate Ansible by assigning roles and variables.
+
+### Monitoring
+
+Collect Puppet, Salt, and Ansible reports and facts. Monitor host configuration, report status, distribution, and trends.
+
+### Orchestration
+
+Utilize Scripts or Ansible playbooks for remote execution or pull them using MQTT.
+
+### Content Management
+
+To mirror and stage content like RPM and DEB packages, Container images, or any type of file have a look at [Katello](https://github.com/Katello), a fixed set of plugins for Foreman.
+
+With a wide range of plugins, you can add even more features, not all of them can be found at this organization, so have a look at the [plugin overview](https://theforeman.github.io/foreman-plugin-overview/).
+
+## Issue tracking
+
+Not all parts of the project track issues here at GitHub! So, if you are missing the issues here, check out the [issue tracker](https://projects.theforeman.org/).
+
+## Getting help
+
+For getting easily started check out the [documentation](https://docs.theforeman.org/) and the [Community support](https://community.theforeman.org/). There are also regular demos on the [YouTube channel](https://www.youtube.com/@Foreman).
```

Comment:
We haven't quite made docs.theforeman.org the official documentation. Most notable, for vanilla Foreman there are a lot of docs missing. I now see we do link all guides for Foreman 3.9 but I don't think that was the intention. I'll get on that.

Reply:
Perhaps not totally ready or finished, but I would already recommend the new docs more for new users, so I would prefer linking to it instead of the manual. But of course I would follow here the official guideline / opinion of the docs team!

Reply:
I think docs.theforeman.org needs a better landing page. Both to describe the state it's in and better navigation. Not limited strictly a blocker to this, but perhaps something you can think about as well.

Reply:
I opened an [issue](https://github.com/theforeman/foreman-documentation/issues/2668) over at foreman-documentation to address this including some suggestions. Is this somewhat similar to what you had mind?

Reply:
Should we address this here or can we merge it? I would like this to land before cfgmgmtcamp.

Reply:
Funny, I also wanted to merge an improved profile before the conferences. So I'm merging this now and we can improve on it further.